debian/postinst

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+MYCROFT=/opt/mycroft
+MYCROFT_LOG=/var/log/mycroft
+MYCROFT_BIN=/opt/mycroft/mycroft-core/bin
+
+mkdir -p $M/skills $MYCROFT_LOG /etc/mycroft
+chown -R root:root $MYCROFT
+chmod -R 777 $MYCROFT_LOG
+
+ln -S $MYCROFT_BIN/mycroft-help /bin/mycroft-help
+ln -S $MYCROFT_BIN/mycroft-cli-client /bin/mycroft-cli
+ln -S $MYCROFT_BIN/mycroft-mic-test /bin/mycroft-test-mic
+ln -S $MYCROFT_BIN/mycroft-msk /bin/mycroft-msk
+ln -S $MYCROFT_BIN/mycroft-msm /bin/mycroft-msm
+ln -S $MYCROFT_BIN/mycroft-pip /bin/mycroft-pip
+ln -S $MYCROFT_BIN/mycroft-say-to /bin/mycroft-say-to
+ln -S $MYCROFT_BIN/mycroft-skill-testrunner /bin/mycroft-skill-testrunner
+ln -S $MYCROFT_BIN/mycroft-speak /bin/mycroft-speak
+ln -S $MYCROFT_BIN/mycroft-stop /bin/mycroft-stop
+ln -S $MYCROFT_BIN/mycroft-start /bin/mycroft-start
+
+cd /opt/mycroft/mycroft-core
+source venv-activate.sh
```

Comment:
You activate the venv but I don't see it being created?

Reply:
This is the venv which is already provided in the mycroft-core repo.
It just cds into the installed mycroft-core repo and creates a venv right there.
And as I see every script in the bin folder do actually activate the venv before doing anything else.

Reply:
Ok so the virtualenv is prebuilt and included in the package then. gotcha.